spec/sms_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'sms'
+require 'takeaway'
+
+describe SMS do
+  subject(:sms) { described_class.new(config, client: client) }
+
+  let(:client) { double(:client, messages: messages) }
+  let(:messages) { double(:messages) }
+# implementing Twilio: https://rails.devcamp.com/trails/rest-microservices/campsites/sms-messages/guides/creating-a-rspec-stub-to-mimic-sending-sms-messages
+  let(:config) do
+    {
```

Comment:
To avoid uploading things that you might wish to keep private: https://www.rubyguides.com/2019/01/ruby-environment-variables/